nncf/quantization/algorithms/smooth_quant/torch_backend.py

```diff
@@ -127,7 +133,12 @@ def get_weight_value(node_with_weight: NNCFNode, model: NNCFNetwork, nncf_graph:
         return Tensor(weight_data)
 
     @staticmethod
-    def weight_update_command(node_with_weight: NNCFNode, weight_value: np.ndarray) -> PTWeightUpdateCommand:
+    def weight_update_command(
+        node_with_weight: NNCFNode, nncf_graph: NNCFGraph, weight_value: torch.Tensor
+    ) -> PTWeightUpdateCommand:
+        if is_experimental_torch_tracing_enabled():
+            weight_node = get_const_node(node_with_weight, node_with_weight.metatype.weight_port_ids[0], nncf_graph)
```

Comment:
You hard coded choosing weight_port_ids by index 0. Does it work for the `mamul` operation?

Reply:
matmul still does not supported